docs/coordinates/skycoord.rst

```diff
@@ -761,7 +761,7 @@ can specify one or more coordinate positions as follows:
 **REPRESENTATION**
 
 The representation can be supplied either as a
-`~astropy.coordinates.representation.BaseRepresentation` class (e.g.,
+`~astropy.coordinates.representation.base.BaseRepresentation` class (e.g.,
```

Comment:
I think this is better changed to `~astropy.coordinates.BaseRepresentation`, since that is where we want users to import from.